src/biogeophys/OzoneMod.F90

```diff
@@ -86,7 +86,7 @@ module OzoneMod
 
   ! TODO(wjs, 2014-09-29) This parameter will eventually become a spatially-varying
   ! value, obtained from ATM
-  real(r8), parameter :: forc_ozone = 100._r8 * 1.e-9_r8  ! ozone partial pressure [mol/mol]
+  !real(r8), parameter :: forc_ozone = 100._r8 * 1.e-9_r8  ! ozone partial pressure [mol/mol]
```

Comment:
- [ ] Please go ahead and remove this commented-out line

Reply:
Can do! Do you want me to also remove your comment/note below about making some of these parameters input parameters? ([here](https://github.com/adrifoster/CTSM/blob/ac07fe0ec88cb5bf7815fe87ca91499be068d382/src/biogeophys/OzoneMod.F90#L91)) Or leave for now?

Reply:
Thanks. I guess I'd leave that for now (though I don't feel strongly).